lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -191,11 +187,7 @@ export default function LMSFilePicker({
       break;
     case 'authorize':
       continueButton = (
-        <Button
-          key="showAuthWindow"
```

Comment:
Can you help me understand the removal of the `key` prop here?

Reply:
My understanding is that it is no longer required because the `buttons` prop of dialog now accepts a single button as an argument, which is set to `continueButton` here, rather than requiring the buttons to be wrapped in an array. When the `buttons` prop is a single JSX element rather than an array, a `key` is not required. This is just a standard Preact/React behavior.

Have I got this right @esanzgar?